src/common/legacy_config_opts.h

```diff
@@ -26,6 +26,7 @@ OPTION(run_dir, OPT_STR)       // the "/var/run/ceph" dir, created on daemon sta
 OPTION(admin_socket, OPT_STR) // default changed by common_preinit()
 OPTION(admin_socket_mode, OPT_STR) // permission bits to set for admin socket file, e.g., "0775", "0755"
 
+OPTION(daemon_debug, OPT_BOOL) // default changed by common_preinit()
```

Comment:
We don't add new options to this file. Please drop this change.

Reply:
Using the `fuse_debug` will be very simple to fix this issue. And the `daemon_debug` is not needed any more. Will remove it.
Thanks